src/lib/utils/number.ts

```diff
@@ -0,0 +1,7 @@
+export function isNumber(input: unknown): input is number {
+  return typeof input === "number" || !isNaN(Number(input));
+}
+
+export function between(value: number, min: number, max: number) {
+  return max <= min ? min : Math.min(max, Math.max(min, value));
```

Comment:
Confusing. The function should `throw` if `max < min`. Then it's just `return value >= min && value <= max`, assuming you want to check if `value` is between or equal `min` and `max`.

Reply:
This is a Quasar function, I just copied it.
I guess they prefer a working API than something that throws. I can change if needed though

Reply:
Not urgent, but the change would make it cleaner imho.

Reply:
I added a console warning if max < min (see b0a295e896ab0877eda6df44d6c0c7648ff398fb). I don't only want to check if `value` is between `min` and `max`, I want to retrieve an actual number:
- if `min >= value || min >= max` --> I take `min`
- if `max <= value` --> I take `max`
- else, if `value` is between `min` and `max` --> I take `value`